src/ontology/mondo-ingest.Makefile

```diff
@@ -160,15 +160,21 @@ mappings: sssom $(ALL_MAPPINGS)
 #################
 # Utils #########
 #################
-# Documentation for this commands in this section is in: `docs/developer/ordo.md`
-
+# Documentation for `repo   rt-mapping-annotations` and `update-jinja-sparql-queries`: `docs/developer/ordo.md`
 report-mapping-annotations:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/report_mapping_annotations.py
 
 update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl $(TMPDIR)/%_relevant_signature.txt
+	python3 ../scripts/exclusion_term_expansion/exclusion_term_expansion.py \
+	--exclusions-path $(word 1,$^) \
+	--onto-path $(TMPDIR)/$(word 2,$^).owl \
+	--relevant-signature-path $(word 3,$^) \
+	--onto-name $*
```

Comment:
@matentzn FYI: Added param `--onto-name` to script. This is good for future proofing. Previously, I was extracting the ontology name from one of the file names that was passed as a param. However, as our pipeline changes, those file names are also subject to change. As such, it is more stable to pass the ontology name explicitly.